# #handling exception error: Try-except
# new_list = [2,4,6, "California"] 

# for element in new_list: 
#     try: 
#         print(element/2) 
#     except: 
#         print("The element is not a number!")


#while break loops 
n=4 
while n>0: 
    print(n) 
    n=n-1
    if n==2:
        break

print("n become 0 (zero)")